berkeley-function-call-leaderboard/openfunctions_evaluation.py

```diff
@@ -91,7 +91,9 @@ def collect_test_cases(test_filename_total, model_name):
 
 
 def generate_results(args, model_name, test_cases_total):
-    retry_limit = 3
+    RETRY_LIMIT = 3
+    RETRY_DELAY = 65  # Delay in seconds
```

Comment:
Curious where is this number 65 coming from? Add more context here if there are references. 
If this number is arbitrary, let's just put 60 seconds for default 1 minute RETRY_DELAY.

Reply:
60s for the timer to complete. But often we find that even with 60 there is a conflict. So 65 is a safe nos

Reply:
Ok sounds good, can you add this as a comment in code for reference?

Reply:
sure